Source/AFError.swift

```diff
@@ -180,6 +180,8 @@ public enum AFError: Error {
         case certificatePinningFailed(host: String, trust: SecTrust, pinnedCertificates: [SecCertificate], serverCertificates: [SecCertificate])
         /// Public key pinning failed.
         case publicKeyPinningFailed(host: String, trust: SecTrust, pinnedKeys: [SecKey], serverKeys: [SecKey])
+        /// Subject PublicKey Info pinning failed
+        case subjectPublicKeyInfoPinningFailed(host: String, trust: SecTrust, pinnedKeys: [String], serverKeys:[SecKey])
```

Comment:
nit: 
- Might want to rename `pinnedKeys` to `pinnedKeyHashes`
- Might want to expose serverKeyHashes as well to allow for easier debugging